glide.lock

```diff
@@ -1,5 +1,5 @@
 hash: 5d4fb27f861fb4a53f3cfafc8916cba71c7038b3438b57c24ffa36f028689cb2
-updated: 2018-03-06T17:28:16.140271-08:00
+updated: 2018-03-14T14:32:50.982036-07:00
```

Comment:
Please move to dep in a separate PR (I created an issue for this).

Reply:
Will do